recipe/meta.yaml

```
@@ -11,45 +15,68 @@ source:
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
-
-requirements:
-  host:
-    - python >=3.10
-    - setuptools >=64
-    - setuptools-scm >=8
-    - pip
-  run:
-    - python >=3.10
-    - numpy >=2.0
-    - requests
-    - scipy
-    # optional, server, but required to pass the import test
-    - docopt-ng
-    - jinja2
-    - webob
-    # optional deps
-    # server
-    # - beautifulsoup4
-    # - lxml
-    # functions
-    # - coards
-    # - gsw
-    # netcdf
-    # - netcdf4
-
-test:
-  imports:
-    - pydap
-  requires:
-    - gunicorn  # [not win]
-    - pastedeploy  # [not win]
-    - pip
-  commands:
-    - pip check
-    - pydap -h  # [not win]
+  number: 1
 
+outputs:
+  - name: "pydap"
+    build:
+      script: {{ PYTHON }} -m pip install . -vv
```

This is an overkill in most recipes but it is necessary in multiple outputs due to a bug in conda-build that ignore those flags and end up bringing pip packages.

```suggestion
      script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation 
```